App.js

```diff
@@ -1,9 +1,29 @@
+import 'react-native-gesture-handler';
 import React from 'react';
-import {StyleSheet, TextInput} from 'react-native';
-import {Home, Login, SignUp} from './src/containers';
+import {NavigationContainer} from '@react-navigation/native';
+import {createStackNavigator} from '@react-navigation/stack';
+import {Home, LogIn, SignUp} from './src/containers';
+
+const Stack = createStackNavigator();
+
+const listScreens = [
+  {name: 'SignUp', component: SignUp},
+  {name: 'LogIn', component: LogIn},
+  // {name: 'Home', component: Home}, 
```

Comment:
Codacy has a fix for the issue: [Trailing spaces not allowed.](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
  // {name: 'Home', component: Home},
```